src/app.py

```diff
@@ -83,10 +85,18 @@ def calculate_deltas(from_tick_number: int, to_tick_number: int, actions_in_tick
     return deltas
 
 
+async def fetch_wrapper(process_data: ProcessData) -> str:
```

Comment:
I don't like the names `fetch_wrapper` and `send_wrapper`. Maybe let's use comething like `fetch_actions_from_process` and `send_scene_to_process`?